go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/openshift/api v0.0.0-20210831091943-07e756545ac1
 	github.com/openshift/build-machinery-go v0.0.0-20210806203541-4ea9b6da3a37
 	github.com/openshift/client-go v0.0.0-20210831095141-e19a065e79f7
-	github.com/openshift/library-go v0.0.0-20210915142033-188c3c82f817
+	github.com/openshift/library-go 86426cb401393369f6de8d4bdef2849dd2d8a228
```

Comment:
should be `github.com/openshift/library-go release-4.9`, makes this a bit easier to review in my opinion